dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportController.java

```diff
@@ -370,6 +370,17 @@ Page<ObjectNode> getTrackedEntityAttributeChangeLog(
         trackedEntityChangeLogService.getTrackedEntityChangeLog(
             trackedEntity, program, operationParams, pageParams);
 
-    return fieldFilterRequestHandler.handle(request, "changeLogs", changeLogs, requestParams);
```

Comment:
Is this `fieldFilterRequestHandler.handle` method unused now?

Reply:
no, is there a reason why it should?

Reply:
No, my point was just to remove it if it is unused.